python/cudf/udf_cpp/groupby/CMakeLists.txt

```
@@ -0,0 +1,92 @@
+# =============================================================================
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+cmake_minimum_required(VERSION 3.23.1)
+
+include(rapids-cmake)
+include(rapids-cuda)
+include(rapids-find)
+
+rapids_cuda_init_architectures(groupby-udf-cpp)
+
+# Create a project so that we can enable CUDA architectures in this file.
+project(
+  groupby-udf-cpp
+  VERSION 23.02.00
+  LANGUAGES CUDA
+)
+
+rapids_find_package(CUDAToolkit REQUIRED)
```

I think we can remove all of this now. The top-level cudf CMakeLists.txt now requires CUDA, so we shouldn't need to independently configure it here.
```suggestion
include(rapids-find)
```